db/init/2023-03-05-color.sql

```diff
@@ -0,0 +1,8 @@
+SET NAMES utf8mb4;
+SET FOREIGN_KEY_CHECKS = 0;
+
+INSERT INTO `system_settings` (strSettingVariable, strDefaultValue, strDescription)
+VALUES ('settingColorPrimary', '#000000', 'Choose a primary color:'),
```

Comment:
The default value should no be black, because if you reinit the project or if you setup a new project, all buttons will be black. Please use the blue color (primary) and the gray color (secondary) from Tabler.